applications/pelion_client/README.rst

```diff
@@ -0,0 +1,516 @@
+
+The following configuration files are specified for each build type:
+
+* :file:`app_${CMAKE_BUILD_TYPE}.conf` - Contains configuration of the application.
+* :file:`mcuboot_${CMAKE_BUILD_TYPE}.conf` - Contains configuration of the bootloader.
+* :file:`pm_static_${CMAKE_BUILD_TYPE}.conf` - Contains configuration of the partition layout.
+
+Additionally, each build target directory contains the following configuration files:
+
+* :file:`mcuboot_private.pem` - Private signature used for binary image signing.
+* :file:`dts.overlay` - The DTS overlay used to adjust the DTS for needs of the application (that is, enable and configure the required hardware).
+* ``_def`` files - Configuration files for application modules.
+
+The :file:`nrf52840dk_nrf52840` directory also includes the following optional file:
+
+* :file:`config-tls-pelion.h` - mbed TLS configuration overlay; used when Zephyr's default mbed TLS version is selected (this is a different mbed TLS version than the one provided by :ref:`nrfxlib:nrf_security`).
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`applications/pelion_client`
+
+Before building and running the firmware, ensure that the Pelion cloud platform is set up.
+Also, the device must be provisioned and configured with the credentials for the connection attempt to succeed.
+See `Configuration`_ for details.
+
+The application is built the same way to any other |NCS| application or sample.
+
+.. include:: /includes/build_and_run.txt
+
+.. note::
+    Information about the known issues can be found in |NCS|'s :ref:`release_notes` and on the :ref:`known_issues` page.
+
+Selecting a build type
+======================
+
+Before you start testing the application, you can select one of the build types, depending on your development kit and building method.
+
+Selecting a build type in SES
+-----------------------------
+
+.. include:: /gs_modifying.rst
+   :start-after: build_types_selection_ses_start
+   :end-before: build_types_selection_ses_end
+
+Selecting a build type from command line
+----------------------------------------
+
+.. include:: /gs_modifying.rst
+   :start-after: build_types_selection_cmd_start
+   :end-before: build_types_selection_cmd_end
+
+Testing
+=======
+
+After programming the application and all the prerequisites to your development kit, test the application by performing the following steps:
+
+1. |connect_kit|
+#. Connect to the kit with a terminal emulator to observe log messages (for debug build types).
+#. Reset the development kit.
+#. Turn on the development kit.
+   **LED1** and **LED2** start blinking, which indicates the network is connecting to the Pelion cloud.
+   After several seconds, both LEDs stop blinking and remain turned on.
+   This indicates that the device has established connection to the network backend and the Pelion server.
+#. Log in to Pelion's Device Management Platform.
```

Comment:
Pelion Device Management Portal

Reply:
There are other instances of this: if we're talking about the frontend, assuming no big product name changes have happened, it's Portal.